internal/pkg/runtime/engine/singularity/rpc/args.go

```diff
@@ -7,6 +7,7 @@ package rpc
 
 import (
 	"os"
+	"syscall"
```

Comment:
Would it make more sense to use `golang.org/x/sys/unix` instead of `syscall`? Since `syscall` is deprecated.

_From the `syscall` godoc_
>Deprecated: this package is locked down. Callers should use the corresponding package in the golang.org/x/sys repository instead. That is also where updates required by new systems or versions should be applied. See https://golang.org/s/go1.4-syscall for more information.

Reply:
Yes but not in this PR, I will open a technical debt issue for that to remove `syscall` from Singularity codebase